.purs-repl

```diff
@@ -0,0 +1 @@
+import Prelude
```

Comment:
can you explain what this does? Is it like a `.bashrc` file for the PureScript REPL? If so, … is the Prelude not included by default there?

Reply:
If it doesn't cause too much pain, I would be in favor of removing this file. Keep the root folder clean.

Reply:
> can you explain what this does? Is it like a .bashrc file for the PureScript REPL? If so, … is the Prelude not included by default there?

Kind of, yes. And no, `Prelude` isn't imported by default, unless you're running `spago repl` outside of a Spago project.

> If it doesn't cause too much pain, I would be in favor of removing this file. Keep the root folder clean.

I wouldn't mind removing it too much. I rarely use the REPL, at least with Insect.